aiidalab_widgets_base/structures.py

```diff
@@ -53,15 +51,11 @@ def __init__(self, importers, storable=True, node_class=None, **kwargs):
 
         self.structure_ase = None
         self._structure_node = None
-
-        self.viewer = nglview.NGLWidget()
-
+        self.viewer = StructureDataVisualizer(downloadable=False)
         self.btn_store = ipw.Button(description='Store in AiiDA', disabled=True)
         self.btn_store.on_click(self._on_click_store)
-
         self.structure_description = ipw.Text(placeholder="Description (optional)")
         self.structure_description.observe(self.reset_structure, names=['value'])
-
         self.data_format = ipw.RadioButtons(options=self.DATA_FORMATS, description='Data type:')
         self.data_format.observe(self.reset_structure, names=['value'])
```

Comment:
I'm a bit lost here. Is the idea here that the user selects what data format they want to upload?

Reply:
Here the user selects format used to store structure in the AiiDA database.

Reply:
I added a comment to clarify this.